rules/bugfix.bzl

```diff
@@ -0,0 +1,23 @@
+def find_cc_toolchain(ctx):
```

Comment:
Missing license header. I think the license checker doesn't know about bzl files yet.

Reply:
Added it and filed an issue: https://github.com/lowRISC/opentitan/issues/9069